lib/cc/analyzer/engine_output_filter.rb

```diff
@@ -22,7 +22,7 @@ def check_disabled?(issue)
       end
 
       def ignore_fingerprint?(issue)
-        @config.fetch("exclude_fingerprints", []).include?(issue.fingerprint)
+        @config.fetch("exclude_fingerprints", []).include?(issue.fingerprint(expect_valid_locations: false))
```

Comment:
`expect_valid_locations ` seems like kind of a weird argument to me.  since this is the only place where it is false, could you just add a rescue block here for the `SourceExtractor::InvalidLocation` instead?  i guess it could just return false in the rescue and error further downstream?